docs/README.md

```diff
@@ -20,6 +20,7 @@ We also maintain an active [list of RFCs (Requests for Comments)](RFC-List.md) w
 
 ### 2.1.1 Deploy with Docker (Recommended)
 * [Deploy with Docker](docker/README.md)
+* [Import data with Docker](docker/import_data.md)
```

Comment:
good to also add to `SUMMARY.md`, this is what gitbook.io uses to make the table of contents (TOC). Ideally the TOC in `README.md` and `SUMMARY.md` are always the same. I should prolly add some test for that